iosApp/SurveyTests/Sources/Specs/Presentation/Modules/SurveySelection/Account/AccountViewDataSourceSpec.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  AccountViewDataSourceSpec.swift
+//  Survey
+//
+//  Created by Bliss on 22/12/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Nimble
+import Quick
+import Shared
+
+@testable import Survey
+
+final class AccountViewDataSourceSpec: QuickSpec {
+
+    override func spec() {
+
+        var logOutUseCase: LogOutUseCaseKMMMock!
+        var coordinator: AccountCoordinatorMock!
+        var accountViewModel: AccountViewModel!
+        var dataSource: AccountView.DataSource!
+        var accountUiModel: AccountUiModel!
+
+        describe("a Account View View Data Source") {
```

Comment:
Looks like there is a redundant word 🤔 
```suggestion
        describe("a Account View Data Source") {
```